package.json

```diff
@@ -9,6 +9,10 @@
     "lint:fix": "eslint ./src/**/*.js App.js --fix"
   },
   "dependencies": {
+    "EventEmitter": "^1.0.0",
```

Comment:
Use fix version here.

Reply:
It would appear that 1.0.0 is final version
https://www.npmjs.com/package/EventEmitter

Reply:
Or probably you mean to substitute ^ to ~?